README.md

```diff
@@ -30,3 +30,19 @@ docker-compose up
 ```
 
 and nwserver will spin up.
+
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply 
```

Comment:
Some grammar issues here: "To install is simply adding this folder to your solution directory." -> "To install it simply add this folder to your solution directory."